### PR TITLE
[WIP] Tutorial on Safe Reinforcement Learning

### PR DESCRIPTION
**Pull request by romesco**
The tutorial is aimed at all people interested in safe reinforcement learning.

It outlines useful techniques in this area and demonstrates how to run baseline algorithms.

### CODE REVIEW

File: Tutorials/safe_reinforcementlearning.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+🚧 Work In Progress
+{: .label .label-yellow}
+
+![this is a dummy header](https://linkedinheaders.com/wp-content/uploads/2018/02/mountain-header.jpg)
+
+## Prerequisites
+
+1. Read [this tutorial](./creating_a_new_tutorial.md) first.
+
+## Safe RL Part 1
```
Comment (yxyang) [inline Tutorials/safe_reinforcementlearning.md:16]: Nice start! Need more content / references.

**Review by yxyang — COMMENTED** [review]
Thanks for the effort. See comment in file.

File: Tutorials/safe_reinforcementlearning.md (hunk 1 above)
Comment (romesco) [reply Tutorials/safe_reinforcementlearning.md:16]: Great! Will do!

**Comment by romesco** [comment]
For the record, this was not a real PR. It was meant to be illustrative for the `creating_a_new_tutorial` tutorial.

**Pull request closed without merging**